app/src/auth/components/UsernamePasswordLogin/index.jsx

```diff
@@ -119,20 +127,24 @@ const UsernamePasswordLogin = ({ onEthereumClick }: Props) => {
                     readOnly
                     hidden
                 />
-                <TextInput
+                <FormControlLabel state={errors.password && 'ERROR'}>
+                    <Translate value="auth.labels.password" />
+                </FormControlLabel>
+                <Text
                     name="password"
                     type="password"
-                    label={I18n.t('auth.labels.password')}
                     value={form.password}
                     onChange={onInputChange(setFormField)}
-                    error={errors.password}
-                    processing={step === 1 && isProcessing}
                     autoComplete="current-password"
                     className={styles.passwordInput}
                     autoFocus
-                    preserveLabelSpace
-                    preserveErrorSpace
                 />
+                <FormControlUnderline
+                    state={(step === 1 && isProcessing && 'PROCESSING') || (errors.password && 'ERROR')}
```

Comment:
Don't feel strongly about it but maybe these "state" strings should be imported?